src/main/java/org/dinosaur/foodbowl/domain/comment/api/CommentController.java

```diff
@@ -22,8 +28,19 @@ public class CommentController {
     @PostMapping
     public ResponseEntity<Void> createComment(
             @Valid @RequestBody CommentCreateRequest commentCreateRequest,
-            @MemberId Long memberId) {
+            @MemberId Long memberId
+    ) {
         commentService.save(memberId, commentCreateRequest);
-        return ResponseEntity.created(URI.create("/posts/" + commentCreateRequest.getPostId())).build();
+        return ResponseEntity.created(URI.create("/api/v1/posts/" + commentCreateRequest.getPostId())).build();
+    }
+
+    @PutMapping("/{commentId}")
+    public ResponseEntity<Void> updateComment(
+            @Positive(message = "댓글 ID는 양수만 가능합니다.") @PathVariable Long commentId,
```

Comment:
```suggestion
            @PathVariable @Positive(message = "댓글 ID는 양수만 가능합니다.") Long commentId,
```
`@PathVariable`을 앞으로 빼는건 어떨까요? 해당 변수가 어디에 쓰이는데 검증을 수행한다의 흐름으로 코드를 읽기 좋은 것 같습니다!